src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_CSharp_Nested_HTML.semantic.txt

```diff
@@ -5,6 +5,7 @@
 0 4 1 73 0 //razorTransition
 0 1 1 73 0 //razorTransition
 0 1 3 15 0 //keyword
+0 0 25 89 0 //razorCodeBackground
```

Comment:
I think it's finally hitting where it's only the `lines` that are offsets and the characters are always absolute indexes for the current line.

Reply:
It is not the case that characters are absolute index for the current line. Specifically it is 

>  deltaStart : token start character, relative to the previous token (relative to 0 or the previous token’s start if they are on the same line)

The confusing part (to me) being that it's relative to the previous token START.